src/minecraft/kotlin/xyz/wagyourtail/unimined/internal/minecraft/patch/forge/CleanroomMinecraftTransformer.kt

```diff
@@ -103,9 +132,15 @@ open class CleanroomMinecraftTransformer(project: Project, provider: MinecraftPr
     }
 
     override fun libraryFilter(library: Library): Library? {
-        if (library.name.startsWith("oshi-project:")) {
+        val name = library.name
+        if (vanillaExcludesSet.contains(name)) {
             return null
         }
+        if (name.startsWith("org.lwjgl") && name.split(":").size > 3) {
+            if (!name.split(":")[2].endsWith(lwjglClassifier)) {
```

Comment:
I don't think this'll ever be hit by the vanilla client json, because it uses `library.natives` and doesn't put them top level

Reply:
I wonder in which step it downloads natives though